test/complex.jl

```diff
@@ -278,8 +276,7 @@ end
 end
 
 @testset "expm1" begin
-    #  expm1(conj(z)) = conj(expm1(z))
-
+    # expm1(conj(z)) = conj(expm1(z))
```

Comment:
Likewise here, perhaps incorporate the comment in the testset description and nix the comment?